src/overlays/actors/ovl_En_Cs/z_en_cs.c

```diff
@@ -416,7 +449,7 @@ void EnCs_Update(Actor* thisx, GlobalContext* globalCtx) {
 }
 
 void EnCs_Draw(Actor* thisx, GlobalContext* globalCtx) {
-    static u64* eyeTextures[] = { 0x06002130, 0x06002930, 0x06003130 };
+    static u64* eyeTextures[] = { gGraveYardKidEyesOpenTex, gGraveYardKidEyesHalfTex, gGraveYardKidEyesClosedTex };
```

Comment:
I think there is no need to have `static` if the variable is inside the function. Also, could you add a trailing comma and re-run `format.sh`?
```suggestion
    u64* eyeTextures[] = { gGraveYardKidEyesOpenTex, gGraveYardKidEyesHalfTex, gGraveYardKidEyesClosedTex, };
```

Reply:
It needs to be `static` otherwise its stored on the stack and not in `data`.